django-verdant/rca/static/rca/css/components/course-booking.less

```diff
@@ -0,0 +1,46 @@
+#course-booking-form {
```

Comment:
is there a reason why you've used id selectors here over classes? I've not seen the markup so if you can only grab it from the id then ignore this : )